crates/bevy_time/src/time.rs

```diff
@@ -122,11 +148,29 @@ impl Time {
     }
 
     /// The time from startup to the last update in seconds
+    ///
+    /// If you intend to cast this to an `f32` value, note that this value is monotonically increasing,
+    /// and that its precision as an `f32` will noticeably degrade over time (in a matter of hours).
+    /// If that precision loss is unacceptable, you should use [`Time::seconds_since_startup_f32_wrapped`],
+    /// which will return the time from startup modulo a wrapping period.
     #[inline]
     pub fn seconds_since_startup(&self) -> f64 {
         self.seconds_since_startup
     }
 
+    /// The time from startup to the last update, modulo the given wrapping period, in seconds.
+    ///
+    /// Time from startup is a monotonically increasing value and so its precision when read as an `f32`
+    /// will noticeably degrade over time, which causes issues for some uses, e.g. shaders.
+    /// This method avoids noticeable degradation by limiting the values to a much smaller range.
+    ///
+    /// The default wrapping period is one hour.
```

Comment:
```suggestion
```

This is a property of `WrapDuration`, not of the method.

Reply:
Sure, but it avoids the need to go to the WrapDuration docs just to know the default value.

Reply:
It doesn't convince me. That defeats the point of organizing the reference docs to match the structure of the API. Within reason, this rule should always be applied. First of all, if in the future the default value will be changed, what are the chances that we will remember to also change the docs here? Also, how would the reader know how to access the default value? You should also specify that, but doing it will just make the description contain even more foreign info, increasing the chances of the first scenario. Finally, the reader should learn about `WrapDuration` in any case, but putting one part of its description somewhere else does certainly not encourage them to do so.

The only cases that I consider being within reason to not follow this rule is when presenting alternatives, since there is a relationship between two items, and when a public item is almost never used in comparison to the wrapper API (e.g. I would describe the behavior of the entity spawning command in `Commands::spawn` rather than in the `Spawn` struct).